HomeController.java

```diff
@@ -71,6 +71,9 @@ public ResponseDto getPolitischeInformation(@RequestParam String mostSpeechesFor
             System.out.println(responseDto);
             String recievedArray[][] = readCSVFileIntoArray();
             System.out.println("Printing the data from two dimensional array" );
+            while()

```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Syntax error on token &#34;(&#34;, Expression expected after this token